### ISSUE TITLE
[doc][html-template] Add sample for howto customize the web page title

### ISSUE THREAD

**Issue by gasolin** [mozilla-neutrino/neutrino-dev#232]
beyond the `appMountId` sample, change `title` is a must do for serious use of neutrino, or the page will always show "Webpack App" at browser window title.

* https://neutrino.js.org/presets/neutrino-preset-web/
* https://neutrino.js.org/middleware/neutrino-middleware-html-template/

Add link to `https://github.com/jantimon/html-webpack-plugin` and `https://github.com/jaketrent/html-webpack-template` would help since neutrino didn't list all customizable options in doc based on `html-webpack-plugin` and `html-webpack-template`.

ex: Change the web page title from `Webpack App` to `My Web App`
```
{
  "neutrino": {
    "options": {
      "html": {
        "title": "My Web App"
      }
    }
  }
}
```

**Comment by eliperelman** [comment]
@gasolin would you be able to submit a PR for this?

**Comment by eliperelman** [comment]
Resolved via #252.
